tests/unit/pages/search.test.tsx

```diff
@@ -0,0 +1,102 @@
+import { ref, reactive } from 'vue';
+import { test, expect } from 'vitest';
+import { mockNuxtImport, mountSuspended } from '@nuxt/test-utils/runtime';
+import SearchPage from '~/pages/search/index.vue';
+
+test('/search test placeholding', async () => {
+  expect(true);
+});
+
+/* TODO: there is bug in the /search/index.vue component that
+ * causes vitest to crash when the page is mounted. The error msg reads:
+ *
+ * `[Vue warn]: Cannot mutate <script setup> binding "results" from Options API.`
+ *
+ * The fix isn't immediately obvious. I can't pinpoint anywhere that "results" is
+ * mutated in on the /search page.
+ *
+ * I am leaving my work on these unfinished tests in this PR. The code that
+ * triggers this bug is commented out below. It might be easier to resolve once we
+ * have implemented unit testing for the `SearchResult` component.
+ */
+
+// const page = await mountSuspended(SearchPage);
+
+// test('/search page can mount', async () => {
+//   expect(page).toBeTruthy();
+// });
```

Comment:
This test was the problematic one mentioned in the PR itself. If this doesn't get fixed before being merged to `staging` i'll create a new issue for it